src/Corax/Queries/SortingMatches/SortingMultiMatch.cs

```diff
@@ -191,11 +189,11 @@ public int Fill(Span<long> matches)
         {
             return _comparer1.FieldType switch
             {
-                MatchCompareFieldType.Sequence or MatchCompareFieldType.Alphanumeric => Fill<ItemSorting<SequenceItem>>(ref this, matches),
-                MatchCompareFieldType.Integer => Fill<ItemSorting<NumericalItem<long>>>(ref this, matches),
-                MatchCompareFieldType.Floating => Fill<ItemSorting<NumericalItem<double>>>(ref this, matches),
-                MatchCompareFieldType.Spatial => Fill<ItemSorting<NumericalItem<double>>>(ref this, matches),
-                MatchCompareFieldType.Score => Fill<ItemSorting<NumericalItem<float>>>(ref this, matches),
+                MatchCompareFieldType.Sequence or MatchCompareFieldType.Alphanumeric => Fill<ItemSorting<Comparers.LegacySortingMatch.SequenceItem>>(ref this, matches),
```

Comment:
Why do we have something called "legacy"?

Reply:
Needs to be removed